ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs

```diff
@@ -238,6 +275,7 @@ private void UpdateTextBlockCore(TextBlock textBlock, bool measureOnly)
             textBlock.FontSize = _fontSize ?? 15;
             textBlock.FontStyle = _fontStyle ?? new FontStyle();
             textBlock.FontWeight = _fontWeight ?? FontWeights.Normal;
+            textBlock.TextDecorations = new TextDecorationCollection(TextDecorationCollection);
 
```

Comment:
Nice one. Didn't even got in mind to try this. I'll check how can I implement it this way